src/Ryujinx.HLE/HOS/Services/Hid/IHidServer.cs

```diff
@@ -1106,6 +1107,18 @@ private void SetNpadJoyAssignmentModeSingleWithDestinationImpl(ServiceCtx contex
             //       If one is found, it returns the npadIdType of the other Npad and a bool.
             //       If not, it returns nothing.
         }
+        [CommandCmif(134)] // 6.1.0+
```

Comment:
Probably missing newline.
```suggestion

        [CommandCmif(134)] // 6.1.0+
```